scripts/index.js

```diff
@@ -0,0 +1,74 @@
+function getRandomInt(min, max) {
+	min = Math.floor(min);
+	max = Math.ceil(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+
+let diceOne = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+let diceTwo = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+
+
+let button = document.body.querySelector("#roll-dice");
+
+
+
+const loadDice = event =>{
+	if(event) {
+		event.preventDefault;
+	}
+	
+
+	let containerOne = document.body.querySelector("#container-1");
+	const diceUno = diceOne.roll();
+
+	let newClass = "img-0"+diceUno;
+
+	let elm = document.getElementById("container-1");
+
+	if(elm.className !== newClass){
+		elm.className = newClass;
```

Comment:
Well done on switching the entire class with a conditional and `Element.className`